### DISCUSSION TITLE
How to remove/uninstall Synology

### DISCUSSION THREAD

**Discussion by Antergosgeek** [Entware/Entware#1057] [Q&A]
Hello,

I followed [https://github.com/Entware/Entware/wiki/Install-on-Synology-NAS](url) but now decided I would like to remove Entware from my NAS, what are the steps that I need to take?

any help would be greatly appreciated.

**Comment by Antergosgeek** [comment]
Anyone? Or is it as easy as disabling the autostart task?

**Comment by adamlogan** [comment]
Struggling with this as well. I want to uninstall because I can no longer log into my Synology Diskstation dashboard. I've removed everything but the auto-start task. Can't seem to figure out how to disable the autostart task from command line via SSH, and I can't do it via webapp cuz I can't log in.

Did you figure something out?

**Reply by Antergosgeek** [reply]
Unfortunately not really. I think deleting ```/volume1/@Entware/opt``` and disabling the Autostart task should do the trick. But since you can't log in at all I am unsure what to really tell you. Sorry
